### PR TITLE
SAK-44776 use the submitNoCheck hidden button like auto-save does to submit at end

### PR DESCRIPTION
**Pull request by ottenhoff**
CC: @dnhutchins submitNoCheck seems like the logical thing to do here to submit without any checks and to get out of the assessment.

### CODE REVIEW

File: samigo/samigo-app/src/webapp/jsf/widget/timerBar/timerbar.js [hunk 1]
```diff
@@ -315,11 +315,10 @@
                   elapsedTime = data[1];
                   lastAid = data[2];
                   if (totalTime === elapsedTime) {
-                      $("[id$=\\:save]")[0].click();
```
Comment (ottenhoff) [inline samigo/samigo-app/src/webapp/jsf/widget/timerBar/timerbar.js:318]: question for @dnhutchins : why is this code starting at line 317 so different from the block of code at line 296?

File: samigo/samigo-app/src/webapp/jsf/widget/timerBar/timerbar.js [hunk 2]
```diff
@@ -315,11 +315,10 @@
...
                       clearInterval(localCount);
                       clearInterval(ajaxCount);
                       $("#timeoutWarning").hide();
-                      leaveAssessment();
+                      $("[id$=\\:submitNoCheck]")[0].click();
```
Comment (bjones86) [inline samigo/samigo-app/src/webapp/jsf/widget/timerBar/timerbar.js:321]: I'm not too familiar with this code. Conceptually it makes sense. Would it be appropriate to move this `submitNoCheck` click into the `leaveAssessment()` routine?

Comment (ottenhoff) [reply samigo/samigo-app/src/webapp/jsf/widget/timerBar/timerbar.js:321]: i think leaveAssessment() is for when the assessment was already submitted by the back-end java timer

**Review by adrianfish — APPROVED** [review]
Looks good to me, using the same button makes sense.

**Review by bjones86 — APPROVED** [review]

**Pull request merged**